pkg/kubectl/run.go

```diff
@@ -209,6 +210,94 @@ func (DeploymentAppsV1Beta1) Generate(genericParams map[string]interface{}) (run
 	return &deployment, nil
 }
 
+type DeploymentAppsV1 struct{}
```

Comment:
During one of our previous sig-cli calls we discussed the chance of deprecating `kubectl run` so that in the long run it'll only run a pod and nothing else. `kubectl create <resource>` will be the path to go. So I'd prefer not to add anything new to run, the create part is ok.